apps/backend/src/location/location.service.ts

```diff
@@ -14,36 +13,61 @@ export class LocationService {
 
   public async getLocationsByID(
     program: Ministries,
-    location_ids: number[],
-    method: LocationMethod
-  ): Promise<LocationEntity[]> {
-    const locationMethod =
-      method === LocationMethod.Bank
-        ? LocationMethod.Bank
-        : Not(LocationMethod.Bank);
-    return await this.locationRepo.find({
+    location_ids: number[]
+  ): Promise<NormalizedLocation[]> {
+    const locations = await this.locationRepo.find({
       where: {
         source_id: program,
-        method: locationMethod,
         location_id: In(location_ids),
       },
       order: {
         location_id: 'ASC',
       },
     });
+    return this.normalizeLocations(locations);
+  }
+  public normalizeLocations(locations: LocationEntity[]): NormalizedLocation[] {
```

Comment:
Comment block

Reply:
done (not showing here as outdated but can be seen in diff...)
![Screenshot 2023-06-06 at 7 08 05 PM](https://github.com/bcgov/PaymentCommonComponent/assets/78890675/54396f83-4919-47bd-af78-20e6cc734334)